right/src/layer.c

```diff
@@ -0,0 +1,11 @@
+#include "layer.h"
+
+uint8_t ActiveLayer = LAYER_ID_BASE;
+
+void Layer_MoveTo(uint8_t layer) {
```

Comment:
I would rather leave out these helper functions. They don't seem to add to the readability but make our internal API more complex.